### PR TITLE
Fix: Analytics front-end typo

### LINKED ISSUE #540: Analytics front-end typo
On our `Analytics` section we have a small typo where we don't have any `%` sign here so we don't understand what's the number 

image

Also change the title from - `Tasks Completion over time` to `Quests Completion over time`

### PR DESCRIPTION
**Pull request by VictoriaAde**
Fix typo where we don't have any % sign.

Also, change the title from -  `Tasks Completion over time` to `Quests Completion over time`.

### Pull Request type

Below are the labels corresponding to the type of changes my PR introduces:

- Bugfix
- Code style update (formatting, renaming)
- Refactoring (no functional changes, no API changes)

Resolves: #540

### CODE REVIEW

**Comment by VictoriaAde** [comment]
Hi @ayushtom, should I attempt to resolve this conflict by removing the changes to package-lock.json from my branch?

**Comment by him2312** [comment]
Hey @VictoriaAde. A small nit-pick here.

I think we can bring these conditions outside as a variable
```
const formattedUniqueParticipants = uniqueVisitors > 0 ? numberWithCommas(uniqueVisitors) : "NA";
```

and then use that variable inside the JSX.
```
{formattedUniqueParticipants}
```
---
The reason this will be helpful is, when the Virtual DOM algorithm checks for any diffing
- it comes across `numberWithCommas` function inside the JSX.
- The thing about functions is - you can't know if something changed without actually running the function.
- Hence the `numberWithCommas` function might end up being called again on each re-render.
- As this involved doing regex checks and string manipulation, it might become slight heavy if this function is used at multiple touch-points across the page.

Can be prioritised in another PR, if it is beyond the scope of this issue.
cc. @ayushtom @fricoben

**Comment by ayushtom** [comment]
@him2312 These are helpful points! We can add this in a seperate PR since you rightly pointed out that this will be beyond the scope of this issue. Can you create an issue with appropriate details for this?

**Comment by VictoriaAde** [comment]
@him2312, that is a good spot.
`numberWithCommas` is being used in more places than you mentioned above, so that would mean we would be creating more variables for all the places `numberWithCommas` is being used. 
I just want to be sure I get you well :)

File: app/analytics/[questId]/page.tsx [hunk 1]
```diff
@@ -191,7 +191,7 @@ export default function Page({ params }: BoostQuestPageProps) {
                     <div className="flex flex-wrap gap-2 items-baseline">
                       <span className={analyticsStyles.highlightedText}>
                         {uniqueVisitors > 0
-                          ? computePercentage(questParticipants)
+                          ? String(`${computePercentage(questParticipants)}%`)
```
Comment (ayushtom) [inline app/analytics/[questId]/page.tsx]: is explicitly converting to `String` neccessary here? template strings works too no?

Comment (VictoriaAde) [reply app/analytics/[questId]/page.tsx]: @ayushtom When I tried using just template strings, I was getting a cannot convert number error but I just used template strings again and it worked fine.

**Review by ayushtom — APPROVED** [review]
lgtm

**Pull request merged**